lib/rubocop/cop/rspec/mixin/css_selector.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Helps parsing css selector.
+      module CssSelector
+        # @param selector [String]
+        # @return [Boolean]
+        # @example
+        #   id?('#some-id') # => true
+        #   id?('.some-class') # => false
+        def id?(selector)
+          selector.start_with?('#')
+        end
+
+        # @param selector [String]
+        # @return [Boolean]
+        # @example
+        #   multiple_specified?('a.cls b#id') # => true
+        #   multiple_specified?('a.cls') # => false
+        def multiple_specified?(selector)
```

Comment:
I think this needs a better name. Like:
`multiple?`
`multiple_selectors?`
`multiple_elements?`
`multiple_matches`?